homeassistant/components/gios/quality_scale.yaml

```diff
@@ -0,0 +1,86 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      No custom actions are defined.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
```suggestion
  config-flow-test-coverage:
    status: todo
    comment: Config flow tests should end with `CREATE_ENTRY` or `ABORT`.
```

Reply:
It is covered by: https://github.com/home-assistant/core/pull/155756